.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@
 # R specific hooks: https://github.com/lorenzwalthert/precommit
 repos:
 -   repo: https://github.com/lorenzwalthert/precommit
-    rev: v0.3.2.9013
+    rev: v0.4.2
```

Comment:
Totally unrelated to this PR but for the fact that the [check failed](https://github.com/ccao-data/ccao/actions/runs/9106212182) due to a super old version.